qcodes/instrument_drivers/QuTech/IVVI.py

```diff
@@ -58,10 +60,24 @@ def __init__(self, name, address, reset=False, numdacs=16, dac_step=10,
         # values based on descriptor
         self.visa_handle.baud_rate = 115200
         self.visa_handle.parity = visa.constants.Parity(1)  # odd parity
+        self.visa_handle.write_termination = ''
+        self.visa_handle.read_termination = ''
 
         self.add_parameter('version',
                            get_cmd=self._get_version)
-        
+
+        self.add_parameter('check_setpoints',
+                           parameter_class=ManualParameter,
+                           initial_value=False,
+                           label='Check setpoints',
+                           vals=Bool())
+
+        self.add_parameter('dac_set_sleep',
```

Comment:
This parameter has no units (I assume 's' but it could also be 'ms') 

I don't know if this has been implemented yet but at some point we discussed having some extra input for a docstring in the parameter @giulioungaretti has this been implemented? If so that would also be a good addition.